src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -59,6 +63,30 @@ TEST_F(Cgroups2Test, ROOT_CGROUPS2_AvailableSubsystems)
   }
 }
 
+
+TEST_F(Cgroups2Test, ROOT_CGROUPS2_DeviceController)
+{
+  namespace devices = cgroups2::devices;
+
+  // Deny all access.
+  devices::Entry entry = devices::Entry::parse("a *:* rwm").get();
+
+  EXPECT_SOME(devices::configure(cgroups2::ROOT_CGROUP, {}, {entry}));
+  Try<vector<uint32_t>> attached = ebpf::cgroups2::attached(
+      cgroups2::ROOT_CGROUP);
+
+  ASSERT_SOME(attached);
+  EXPECT_EQ(1, (int) attached->size());
+
+  // Verify that we can't open /dev/null.
+  EXPECT_ERROR(os::open(os::DEV_NULL, O_RDWR));
+
+  EXPECT_SOME(ebpf::cgroups2::detach(cgroups2::ROOT_CGROUP, attached.get()[0]));
```

Comment:
Another reason we don't want to modify the root cgroup, if our test fails or is killed, we'll leave our device program attached?

Reply:
Will update to create a new cgroup for testing.